compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1457,6 +1469,22 @@ LogicalResult OnlineAttentionOp::verify() {
       return failure();
   }
 
+  auto &block = attnOp.getRegion().front();
+  auto blockTys = block.getArgumentTypes();
+  if (blockTys.size() != 1) {
+    return attnOp->emitOpError("expected one block arguments");
+  }
```

Comment:
Why does this region only allow 1 arguement?

Reply:
The only argument is the score. In the future we could expand for including the batch and indices args but that is mostly used for masking which we already support.

Reply:
I'm a bit confused. I was under the impression that the region allows an elementwise operation on the score. This only allows a unary operation on the score. Scaling the score by a tensor is something that should be supported by this region, right?